src/pyhf/optimize/autodiff.py

```diff
@@ -1,9 +1,32 @@
+"""Helper Classes for use of automatic differentiation."""
 import scipy
 from .. import get_backend
 
 
 class AutoDiffOptimizerMixin(object):
-    def minimize(self, objective, data, pdf, init_pars, par_bounds, fixed_vals=None):
+    """Mixin Class to build optimizers that use automatic differentiation."""
+
+    def __init__(*args, **kwargs):
```

Comment:
we don't need to define the `__init__` if you're not changing it.

Reply:
thought so at first as well but it's needed bkg as a mixin, arg,kwarg can be filled and if super().__init__ is called on `object` it balks

Reply:
oh, right, good point... we should just delete the `pass` line or have codefactor ignore it...